doc/engineering-reference/src/building-system-simulation-system-manager/plant-condenser-loops.tex

```diff
@@ -397,8 +397,8 @@ \subsubsection{SequentialLoad}%
 25              & 25            & 0            & 0                             & 0.625      & -           \\ \hline
 50              & 40            & 10           & 0                             & 1 **       & 0.15 *      \\ \hline
 100             & 40            & 60           & 0                             & 1 **       & 0.6         \\ \hline
-150             & 40           & 100           & 0                             & 1 **       & 1 **        \\ \hline
-200             & 40           & 100           & 50                            & 1 **       & 1 **        \\ \hline
+150             & 40           & 100           & 10                            & 1 **       & 1 **        \\ \hline
```

Comment:
Thanks for the catch, I forgot to update this table after I changed the Capacity of the equipment